### PR TITLE
compute input file for mineos

### PR DESCRIPTION
**Pull request by sannecottaar**

### CODE REVIEW

**Comment by ian-r-rose** [comment]
Do we need to include the output file? For the most part we have not included the outputs of example scripts.

File: examples/example_write_mineos_input.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+
+"""
+_________________________________________________________________________
```
Comment (tjhei) [inline examples/example_write_mineos_input.py]: remove lines

File: examples/example_write_mineos_input.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+# Import BurnMan
+sys.path.insert(1, os.path.abspath('../'))  # add path to burnman
+import burnman
+from burnman import minerals  # import mineral library seperately
+
+
+def write_mineos_input(rock, min_depth=670.e3, max_depth=2890.e3, name='burnmantestrock',
+                       mineos_ref='prem_noocean', plotting=False):
+    """
+    _________________________________________________________________________
```
Comment (tjhei) [inline examples/example_write_mineos_input.py]: remove lines

File: examples/example_write_mineos_input.py [hunk 3]
```diff
@@ -0,0 +1,199 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+
+"""
+_________________________________________________________________________
+This example script produces an input file to compute normal modes in
+Mineos for a lower mantle with pyrolitic or chondritic composition.
```
Comment (tjhei) [inline examples/example_write_mineos_input.py]: reference to Mineos? Where can one download it?

File: examples/example_write_mineos_input.py [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    # convert depths to pressures
+    pressures = burnman.seismic.PREM().pressure(depthrange)
+
+    # Computing adiabatic temperatures. T0 is a choice!
+    T0 = 1900  # K
+    temperatures = burnman.geotherm.adiabatic(pressures, T0, rock)
+    # An alternative is the Brown+Shankland (1981)
+    # geotherm for mapping pressure to temperature.
+    # To use this include the line below.
+    #temperature = burnman.geotherm.brown_shankland(pressure)
```
Comment (tjhei) [inline examples/example_write_mineos_input.py]: can we delete this?

File: examples/example_write_mineos_input.py [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+if __name__ == "__main__":
+    # We'll compute the velocities for different compositions at 20 points
+    # within the lower mantle. Here we define the array of those depths.
+    depths = np.linspace(750e3, 2700e3, 20)
+    # We use PREM to convert these depths to pressure values.
+    [pressures] = burnman.seismic.PREM().evaluate(['pressure'], depths)
+
+    #-Defining the rock-
+    #The object burnman.Composite expects two lists, one with the minerals
```
Comment (tjhei) [inline examples/example_write_mineos_input.py]: delete please

**Pull request merged**